src/FSharp.Editing.VisualStudio.v2015/FSharp.Editing.VisualStudio.v2015.fsproj

```diff
@@ -7,25 +7,22 @@
     <SchemaVersion>2.0</SchemaVersion>
     <ProjectGuid>bb0c6f01-5e57-4575-a498-5de850d9fa6c</ProjectGuid>
     <OutputType>Library</OutputType>
-    <RootNamespace>FSharpVSPowerTools.Logic.VS2015</RootNamespace>
+    <RootNamespace>FSharp.Editing.VisualStudio.v2015</RootNamespace>
     <AssemblyName>FSharp.Editing.VisualStudio.v2015</AssemblyName>
     <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
     <TargetFSharpCoreVersion>4.4.0.0</TargetFSharpCoreVersion>
     <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
     <Name>FSharp.Editing.VisualStudio.v2015</Name>
-    <TargetFrameworkProfile />
-    <ReferencePath>
-    </ReferencePath>
   </PropertyGroup>
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
     <DebugSymbols>true</DebugSymbols>
     <DebugType>full</DebugType>
     <Optimize>false</Optimize>
     <Tailcalls>false</Tailcalls>
-    <OutputPath>bin\Debug\</OutputPath>
+    <OutputPath>bin\$(Configuration)</OutputPath>
```

Comment:
those can be moved out of this propertygroup to the previous to remove duplication, same for the release one.